src/readqps.jl

```diff
@@ -124,6 +127,14 @@ function row_type(rtype::String)
     end
 end
 
+# Variable types
+const VTYPE_M = -1  # Marked integer
+const VTYPE_C =  0  # Continuous variable
+const VTYPE_B =  1  # Binary variable
+const VTYPE_I =  2  # Integer variable
+const VTYPE_S =  3  # Semi-continuous
+const VTYPE_N =  4  # Semi-integer
```

Comment:
Are these values in the MPS "standard" somehow or did you make them up? Perhaps using symbols would be somewhat more explicit.

Reply:
The integer values themselves are made-up.
The significance matches Gurobi's [variable types](https://www.gurobi.com/documentation/9.0/refman/vtype.html).

I added the "marked integer" category to make the parsing of bounds easier (see my comments in #28 regarding how solvers have different conventions even for that).

Reply:
Symbols, or `Char`s would indeed make it easier for people to work with.

Reply:
Maybe use a Dict{Symbol,Int} that maps human-readable symbols to the Gurobi ints?

Reply:
A better option would be to make this an `Enum`?
```Julia
@enum(
    VariableType,
   VTYPE_ MARKED_INTEGER,
   VTYPE_ CONTINUOUS,
   VTYPE_ BINARY,
  ...
)
```

Reply:
The issue I have with `Enum`s is that users then have to work with the EnumType, and most likely have to translate it.
I think `Char`s are more flexible and easier to work with. Also, it doesn't hurt that a number of solvers follow a similar convention ('C' for continuous, 'B' for binary, etc...).

Reply:
`Cchar`? Following Gurobi/CPLEX convention is probably okay.